src/cc/usdt/usdt.cc

```diff
@@ -295,24 +295,46 @@ Probe *Context::get(const std::string &provider_name,
 
 bool Context::enable_probe(const std::string &probe_name,
                            const std::string &fn_name) {
+  return enable_probe("", probe_name, fn_name);
+}
+
+bool Context::enable_probe(const std::string &provider_name,
+                           const std::string &probe_name,
+                           const std::string &fn_name) {
   if (pid_stat_ && pid_stat_->is_stale())
     return false;
 
-  // FIXME: we may have issues here if the context has two same probes's
-  // but different providers. For example, libc:setjmp and rtld:setjmp,
-  // libc:lll_futex_wait and rtld:lll_futex_wait.
+  unsigned int matches = 0;
   Probe *found_probe = nullptr;
   for (auto &p : probes_) {
     if (p->name_ == probe_name) {
-      if (found_probe != nullptr) {
-         fprintf(stderr, "Two same-name probes (%s) but different providers\n",
-                 probe_name.c_str());
-         return false;
+      if (found_probe == nullptr && provider_name == "")
+      {
```

Comment:
Please use consistent formatting as rest of the file, or you can just do `clang-format -i`.

Reply:
Will do! I wasn't aware of that tool. I found other issues in this file, I'll ignore those and just fix the ones related to my changes.

Reply:
I've run the tool, and found a helpful script that was able to do it on just the diff:

`git diff -U0 --no-color HEAD^ | clang-format-diff -p1`

Is there an existing document with contributor guidelines that I missed where this could be added?